src/azure-cli/HISTORY.rst

```diff
@@ -45,6 +45,10 @@ Release History
 * [Breaking change] Rename '--max-billing' parameter to '--max-price', for both VM and VMSS, to be consistent with Swagger and Powershell cmdlets
 * vm monitor log show: support query log over linked log analytics workspace.
 
+**Install**
```

Comment:
move this release note to the top, and under it put 2.0.77 so new PRs merged after 2.0.77 build will belong to the release notes of 2.0.78.